app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+        self.__class__.people[self.name] = self
+
+
+def create_person_list(people_dicts: list) -> list:
+    people_instances = [
+        Person(person_dict["name"], person_dict["age"])
+        for person_dict in people_dicts
+    ]
+
+    for person_dict, person_instance in zip(people_dicts, people_instances):
+        if "wife" in person_dict and person_dict["wife"] is not None:
+            person_instance.wife = Person.people[person_dict["wife"]]
+
+        if "husband" in person_dict and person_dict["husband"] is not None:
+            person_instance.husband = Person.people[person_dict["husband"]]
```

Comment:
use get() method instead of parenthesis everywhere for getting value by key to avoid KeyError